tests/raw_window_handle.rs

```diff
@@ -92,25 +92,33 @@ mod raw_window_handle_test {
     #[test]
     fn get_macos_handle() {
         let window = new_hidden_window();
-        match window.raw_window_handle() {
+        let window_handle = match window.window_handle() {
+            Ok(v) => v,
+            Err(err) => panic!(
+                "Received error while getting window handle for MacOs: {:?}",
+                err
+            ),
+        };
+        match window_handle.as_raw() {
             RawWindowHandle::AppKit(macos_handle) => {
-                assert_ne!(
-                    macos_handle.ns_window, 0 as *mut libc::c_void,
-                    "ns_window should not be null"
-                );
-                assert_ne!(
-                    macos_handle.ns_view, 0 as *mut libc::c_void,
-                    "nw_view should not be null"
-                );
-                println!("Successfully received macOS RawWindowHandle!");
+                if macos_handle.ns_view.as_ptr().is_null() {
+                    assert!(false, "ns_view should not be null");
```

Comment:
This would be a lot cleaner if you put the body of the if statement in the assert.